HelloWorld.html

```diff
@@ -5,6 +5,6 @@
     <title>Hello World</title>
   </head>
   <body>
-	<h1>Hello World!<h1>
+	<h1>Bonjour Mode<h1>
```

Comment:
Change Bonjour Mode to Bonjour Mode (spelling error) (Student A - Thomas Lawrence) (Step 6)